components/org.wso2.carbon.identity.media.endpoint/src/gen/java/org/wso2/carbon/identity/media/endpoint/factories/ContentApiServiceFactory.java

```diff
@@ -0,0 +1,30 @@
+/*
+* Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+package org.wso2.carbon.identity.media.endpoint.factories;
+
+import org.wso2.carbon.identity.media.endpoint.ContentApiService;
+import org.wso2.carbon.identity.media.endpoint.impl.ContentApiServiceImpl;
+
+public class ContentApiServiceFactory {
```

Comment:
change to MediaApiServiceFactory

Reply:
answered at: https://github.com/wso2/identity-media/pull/14#discussion_r444845649